src/components/views/JobInfo.js

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+
+import {
+  AccordionSet,
+  Button,
+  Col,
+  ExpandAllButton,
+  KeyValue,
+  Layout,
+  Pane,
+  Row,
+} from '@folio/stripes/components';
+import { TitleManager } from '@folio/stripes/core';
+import { Spinner } from '@folio/stripes-erm-components';
+import ErrorLogs from './ErrorLogs';
+import InfoLogs from './InfoLogs';
+import FormattedDateTime from '../FormattedDateTime';
+
+export default class JobInfo extends React.Component {
+  static propTypes = {
+    data: PropTypes.shape({
+      job: PropTypes.object,
+    }),
+    isLoading: PropTypes.bool,
+    onClose: PropTypes.func.isRequired,
+  };
+
+  state = {
+    sections: {
+      errorLogs: false,
+      infoLogs: false,
+    }
+  }
+
+  renderLoadingPane = () => {
+    return (
+      <Pane
+        defaultWidth="45%"
+        dismissible
+        id="pane-view-job"
+        onClose={this.props.onClose}
+        paneTitle={<FormattedMessage id="ui-local-kb-admin.loading" />}
+      >
+        <Layout className="marginTop1">
+          <Spinner />
+        </Layout>
+      </Pane>
+    );
+  }
+
+  getSectionProps = (id) => {
+    return {
+      id,
+      onToggle: this.handleSectionToggle,
+      open: this.state.sections[id],
+    };
+  }
+
+  handleSectionToggle = ({ id }) => {
+    this.setState((prevState) => ({
+      sections: {
+        ...prevState.sections,
+        [id]: !prevState.sections[id],
+      }
+    }));
+  }
+
+  handleAllSectionsToggle = (sections) => {
+    this.setState({ sections });
+  }
+
+  render() {
+    const { data: { job }, isLoading } = this.props;
+
+    if (isLoading) return this.renderLoadingPane();
+    const isJobQueued = get(job, 'status.value') === 'queued';
```

Comment:
It's kind of tricky to read the flow of this render function when you always have to negate this variable name in your head. All usages of this variable are as `!isJobQueued`, so it might be better to have a `const isJobNotQueued = get(job, 'status.value') !== 'queued'` instead.